packages/react/src/components/card-analytics/CardAnalytics.scss

```diff
@@ -13,7 +13,7 @@
 
   .overflow-menu {
     position: absolute;
```

Comment:
class `overflow-menu` sets to the CardMenu component. Maybe it should moved to the CardMenu.scss?

Reply:
We have different styles for this class. It depends on the container.